opensquirrel/parser/libqasm/parser.py

```diff
@@ -126,12 +126,12 @@ def _get_expanded_instruction_args(self, instruction: cqasm.semantic.Instruction
         """
         expanded_args = self._get_instruction_operands(instruction)
         if isinstance(instruction, cqasm.semantic.GateInstruction):
-            gate_parameter = self._get_named_gate_param(instruction.gate)
+            gate_parameters = self._get_named_gate_parameters(instruction.gate)
         else:
-            gate_parameter = self._ast_literal_to_ir_literal(instruction.parameter)
-        if gate_parameter:
+            gate_parameters = [self._ast_literal_to_ir_literal(parameter) for parameter in instruction.parameters]
+        if gate_parameters:
```

Comment:
```suggestion
        if len(gate_parameters) > 0:
```

Before, `gate_parameter` was an optional. Now, it will always be a list, and it could be empty but never `None` (so `if gate_parameters:` check would always pass).

Reply:
OK, my bad, then just leave it as it is. Thanks for the explanation.